bytes/bytes/api/router.py

```diff
@@ -161,7 +161,7 @@ async def create_raw(
         if meta_repository.has_raw(meta, parsed_mime_types):
             return RawResponse(status="success", message="Raw data already present")
 
-        data = await request.body()
+        data = request.body()
```

Comment:
Same here?

Reply:
I have wrapped this with `async_to_sync` because all the other code in the function also calls sync code.